lib/diagnostic.js

```diff
@@ -6,30 +6,39 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (str) {
+  return paragraph.toUpperCase().split(' ')
```

Comment:
You aren't defining `paragraph` anywhere, make sure you're using the correct variables!